packages/ui/src/validators/components/ValidatorsList.tsx

```diff
@@ -19,8 +19,10 @@ export const ValidatorsList = ({ validators }: ValidatorsListProps) => {
         <ListHeader>Validator</ListHeader>
         <ListHeader>Verification</ListHeader>
```

Comment:
Please add a tooltip next to the "Verification" column header: ![image](https://github.com/Joystream/pioneer/assets/6571453/97be8fe3-5e03-402d-aefe-f1706b3dbd83)

The copy could be something like:

> The profile of Verified validator has been entirely verified by the [Membership working group](https://pioneerapp.xyz/#/working-groups/membership).

But it would be better to have the link point at the "About" tab and add the Working group description to this tab. See #4654

WDYT @dmtrjsg ?

Reply:
Yes its a good idea, and we could just add the description to the "About" section which is exactly the same as the are on the working group cards on the WG landing page where they are shown as a list.